### PR TITLE
Q&M: Fix broken links identified by the Broken links report - fix 20 files

### PR DESCRIPTION
**Pull request by Daidihuang**
Hi @cmastr, @Rick-Anderson, @wadepickett, @tdykstra I have fixed the broken links, please verify, thanks。
https://dev.azure.com/msft-skilling/Content/_workitems/edit/335393/?view=edit

### CODE REVIEW

File: aspnet/ajax/cdn/overview.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ msc.type: content
 >
 > The Microsoft Ajax CDN has no SLA above and beyond using an Azure CDN.
 >
-> Use [this GitHub issue](https://github.com/dotnet/AspNetDocs/issues/116) to report problems with the Microsoft Ajax CDN.
+> Use this GitHub issue to report problems with the Microsoft Ajax CDN.
```
Comment (Rick-Anderson) [inline aspnet/ajax/cdn/overview.md:20]:
```suggestion
> Use [this GitHub issue](https://github.com/dotnet/AspNetCore.Docs/issues/34049) to report problems with the Microsoft Ajax CDN.
```

File: aspnet/mvc/overview/getting-started/database-first-development/generating-views.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ In this tutorial, you:
 ## Prerequisite
 
 * [Create the web application and data models](creating-the-web-application.md)
-* When using Visual Studio 2022, install **Net Framework project and item templates**. For more information, see [this GitHub issue](https://github.com/dotnet/AspNetDocs/issues/591).
+* When using Visual Studio 2022, install **Net Framework project and item templates**. For more information, see this GitHub issue.
```
Comment (Rick-Anderson) [inline aspnet/mvc/overview/getting-started/database-first-development/generating-views.md:31]:
```suggestion
* When using Visual Studio 2022, install **Net Framework project and item templates**. 
```

File: aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-cs.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ Happy Programming!
 
 For more information on the topics discussed in this tutorial, refer to the following resources:
 
-- [ASP.NET Page Lifecycle Diagram](http://emanish.googlepages.com/Asp.Net2.0Lifecycle.PNG)
+- ASP.NET Page Lifecycle Diagram
```
Comment (Rick-Anderson) [inline aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-cs.md:231]:
```suggestion
```

File: aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-vb.md [hunk 4]
```diff
@@ -228,7 +228,7 @@ Happy Programming!
 
 For more information on the topics discussed in this tutorial, refer to the following resources:
 
-- [ASP.NET Page Lifecycle Diagram](http://emanish.googlepages.com/Asp.Net2.0Lifecycle.PNG)
+- ASP.NET Page Lifecycle Diagram
```
Comment (Rick-Anderson) [inline aspnet/web-forms/overview/older-versions-getting-started/master-pages/specifying-the-master-page-programmatically-vb.md:231]:
```suggestion
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]
You can't just delete bad links, you need to delete the text too.

**Comment by Daidihuang** [comment]
> You can't just delete bad links, you need to delete the text too.

Got it, thanks for informing,

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**